modules/services/urxvtd.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.urxvtd;
+
+in
+
+{
+  options = {
+    services.urxvtd = {
+      enable = mkEnableOption "urxvtd";
+
+      package = mkOption {
```

Comment:
I was just about to merge this when I realized that `programs.urxvt.package` also exists and I think it would be quite unfortunate two have two such options. I would suggest removing this option and instead using `config.programs.urxvt.package` and adding
```
assertions = [
  {
    assertion = config.programs.urxvt.enable;
    message = "The urxvtd service module requires"
      + " 'programs.urxvt.enable = true'.";
  }
];

```

Reply:
Agreed. However `programs.urxvt.enable` writes configuration to `xresources.properties`, which is probably more intrusive than we want here. Not sure what the best way to solve that conflict is :)

Reply:
Maybe I'm confused about the purpose of urxvtd. Isn't it some daemon that the urxvt program communicates with? That is, to use urxvtd you would also need to use urxvt?

Reply:
nope, it's optional. the idea is to open terminals faster by forking them off a user daemon, rather than spawning them independently. it's simlar to the emacs daemon in that way, if that's helpful for you :)